gulpfile.js

```diff
@@ -0,0 +1,25 @@
+var gulp = require("gulp");
+var jshint = require("gulp-jshint");
+var mocha = require("gulp-mocha");
+
+var paths = {
```

Comment:
@monkeyFeathers nice!  good call putting your paths in an object for later reference!